browser/ui/brave_browser_command_controller.cc

```diff
@@ -117,8 +123,14 @@ void BraveBrowserCommandController::UpdateCommandForBraveAdblock() {
 }
 
 void BraveBrowserCommandController::UpdateCommandForTor() {
-  UpdateCommandEnabled(IDC_NEW_TOR_CONNECTION_FOR_SITE, true);
-  UpdateCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR, true);
+  const bool isTorEnabled =
+#if BUILDFLAG(ENABLE_TOR)
+      !tor::TorProfileService::IsTorDisabled();
+#else
+      false;
+#endif
+  UpdateCommandEnabled(IDC_NEW_TOR_CONNECTION_FOR_SITE, isTorEnabled);
```

Comment:
why would we call these at all if `BUILDFLAG(ENABLE_TOR)` is false? I think the original code should have been wrapped in `#if BUILDFLAG(ENABLE_TOR)` and then `true` can be replaced by `!tor::TorProfileService::IsTorDisabled()`

Reply:
Done and wrapped whole this method in BUILDFLAG.